sdk/sdk-view/src/test/java/mil/emp3/api/KMLExportTest.java

```diff
@@ -320,14 +322,17 @@ public void exportFailed(Exception ex)
                                         }
                                       });
 
-        while(!resultFound[0]) {
+        int i = 0;
+        while(!resultFound[0] && ++i <= 10) {
             try {
                 Thread.sleep(50);
             } catch (InterruptedException ie) {
 
             }
         }
-        Assert.assertTrue(testPassed[0]);
+        if (i >= 10) {
+            fail("No result in half second for exportEllipse");
```

Comment:
Instead of pausing on exportEllipse will it now just fail sometimes?

Reply:
yes